crates/ruff_python_ast/src/symbol_imports.rs

```diff
@@ -0,0 +1,57 @@
+use std::fmt;
+
+/// A representation of an individual symbol imported via any import statement.
+pub enum AnyImport<'a> {
+    Import(Import<'a>),
+    ImportFrom(ImportFrom<'a>),
+}
+
+/// A representation of an individual symbol imported via an `import` statement.
+pub struct Import<'a> {
```

Comment:
Nit: It's best practice to implement `Debug` and `Clone` for all public types (and other common traits if appropriate)

https://rust-lang.github.io/api-guidelines/interoperability.html#types-eagerly-implement-common-traits-c-common-traits